sensor_msgs/Image image
---
orvis/BoundingBoxes objects